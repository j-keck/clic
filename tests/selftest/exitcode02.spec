# clic --cmdline 'sh -c "exit 1"' --expected-exit-code 2

