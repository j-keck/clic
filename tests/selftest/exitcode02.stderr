Failure - unexpected exit code - expected: 2, actual: 1
